### DISCUSSION TITLE
Add play count/view date to Watch History

### DISCUSSION THREAD

**Discussion by OJ7** [libre-tube/LibreTube#5023] [Ideas]
### Describe your suggested feature

When navigating to Watch History, each video should show how many times I (re)watched it and when I last watched it.

Having a sort by Last Watched (default) and Play Count would be nice to quickly find older videos you're searching for. 

### Other details

I'm not sure if this is something Piped API supports this, but would be nice to have stored offline in LibreTube  regardless. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by IndusAryan** [comment]
>When navigating to Watch History, each video should show how many times I (re)watched it and when I last watched it.

just why???

**Comment by OJ7** [comment]
> > When navigating to Watch History, each video should show how many times I (re)watched it and when I last watched it.
> 
> just why???

> Having a sort by Last Watched (default) and Play Count would be nice to quickly find older videos you're searching for.

**Comment by Bnyro** [comment]
> I'm not sure if this is something Piped API supports this
>
Piped does not support syncing watch history at all at the moment.

Converting to a discussion in order to gain more opinions or upvotes first.

**Comment by RuthlessRuler** [comment]
I second for storing play count.
